src/bin/am/main.rs

```diff
@@ -27,6 +28,8 @@ async fn main() {
         std::process::exit(1);
     }
 
+    tokio::task::spawn(updater::update_check());
```

Comment:
We should also allow for a flag to be set (with a envvar) so that the background check is ignored.